lib/player.js

```diff
@@ -4071,6 +4071,24 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       delete config['streaming']['autoLowLatencyMode'];
     }
 
+    // Deprecate 'manifest.dash.ignoreSupplementalCodecs' configuration.
+    if (config['manifest'] && config['manifest']['dash'] &&
+        'ignoreSupplementalCodecs' in config['manifest']['dash']) {
+      shaka.Deprecate.deprecateFeature(5,
+          'manifest.dash.ignoreSupplementalCodecs configuration',
+          'Please Use manifest.ignoreSupplementalCodecs instead.');
+      delete config['manifest']['dash']['ignoreSupplementalCodecs'];
+    }
+
+    // Deprecate 'manifest.hls.ignoreSupplementalCodecs' configuration.
+    if (config['manifest'] && config['manifest']['hls'] &&
+        'ignoreSupplementalCodecs' in config['manifest']['hls']) {
+      shaka.Deprecate.deprecateFeature(5,
+          'manifest.hls.ignoreSupplementalCodecs configuration',
+          'Please Use manifest.ignoreSupplementalCodecs instead.');
+      delete config['manifest']['hls']['ignoreSupplementalCodecs'];
+    }
```

Comment:
Shouldn't we also set `manifest.ignoreSupplementalCodecs` in that case?

Reply:
This is the behavior we will have from now on anyway (always ignore). As this is opt-in option, I don't see a problem with this.

Reply:
Added then!